Packs/EclecticIQ/ReleaseNotes/1_0_2.md

```diff
@@ -1,3 +0,0 @@
-#### Integrations
-##### EclecticIQ Platform
-- Updated the Docker image to: *demisto/python:2.7.18.24066*.
```

Comment:
Please revert this change. The release notes are already in production and deletion of it is not allowed.
Another option is deprecating the Integration and add the new one as v2.

Reply:
Done